packages/contracts-bedrock/deploy/019-SystemDictatorInit.ts

```diff
@@ -176,7 +176,7 @@ const deployFn: DeployFunction = async (hre) => {
   if (
     (await SystemDictatorProxy.callStatic.admin({
       from: ethers.constants.AddressZero,
-    })) !== hre.deployConfig.controller
```

Comment:
This is an unrelated change, please remove it and propose in its own PR

Reply:
moved to a new PR https://github.com/ethereum-optimism/optimism/pull/5993